lib/activerecord_spanner_adapter/connection.rb

```diff
@@ -193,7 +202,7 @@ def truncate table_name
     end
 
     def self.database_path config
-      "#{config[:project]}/#{config[:instance]}/#{config[:database]}"
+      "#{config[:emulator_host]}/#{config[:project]}/#{config[:instance]}/#{config[:database]}"
```

Comment:
This makes sure that not the same `Spanner` instance is used if the emulator host (or port!) has changed between two tests.